common/src/test/java/com/alibaba/nacos/common/cache/builder/CacheBuilderTest.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
license seems not right. 

Can you check use src/resources/copyright in nacos source code to replace this copyright?

Reply:
done